File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h;

    //prompt for height
    do
    {
        h = get_int("Height: ");
    }
    while (h < 1 || h > 8);


    //draw pyramids
    for (int i = 1; i <= h; i++)
    {
        //determine number of spaces
        int j = h - i;

        //print spaces
        while (j > 0)
        {
            printf(" ");
            j--;
        }

        //print hashes 1
        for (int k = 0; k < i; k++)
        {
            printf("#");
        }

        //print two spaces
        printf("  ");

        //print hashes 2
        for (int k = 0; k < i; k++)
        {
            printf("#");
        }

        //new line
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
import cs50


def main():
    height = 0

    while height < 1 or height > 8:
        height = cs50.get_int("Height: ")

    draw(height)


def draw(n):
    for i in range(1, n + 1):
        print(" " * (n - i), end="")
        print("#" * i, end="")
        print("  ", end="")
        print("#" * i)


if __name__ == "__main__":
    main()
